src/balena.sbvr

```diff
@@ -250,6 +250,7 @@ Term: vpn address
 -- Complex terms
 
 Term: application type
+Term: architecture
```

Comment:
I think it'd make sense to name this as `cpu architecture` since `architecture` is potentially generic and could clash

Reply:
@Page- IMO architecture is fine, since the same can be said for `application`, `device` etc. I don't have a strong opinion though, just trying to think through whether the added verbosity helps in any way, and just `architecture` feels more consistent with the rest of the resource naming. What do you think? How else would we use the term `architecture` in the DB, for example?

Reply:
I would say architecture is more generic, it could be used to refer to the network architecture of the device, the architecture of the device's PCB and how buses/etc are laid out, even "CPU architecture" can mean either the CPU architecture in terms of NUMA nodes/etc or the instruction set architecture, which is what we care about for this resource. Generally starting off with a more specific name is going to reduce the number of support questions that might come up around what "architecture" we're referring to and also saves the need to rename it later

Reply:
FWIW I agree with Page that "cpu architecture" is the proper term here -- it is a term of art.

Reply:
Ok, you convinced me :)

Reply:
@Page- can I get a re-review please?